ui/checkboxradio.js

```diff
@@ -0,0 +1,298 @@
+/*!
+ * jQuery UI Checkboxradio @VERSION
+ * http://jqueryui.com
+ *
+ * Copyright 2014 jQuery Foundation and other contributors
+ * Released under the MIT license.
+ * http://jquery.org/license
+ *
+ * http://api.jqueryui.com/checkboxradio/
+ */
+( function( factory ) {
+	if ( typeof define === "function" && define.amd ) {
+
+		// AMD. Register as an anonymous module.
+		define( [
+			"jquery",
+			"./core",
+			"./widget"
+		], factory );
+	} else {
+
+		// Browser globals
+		factory( jQuery );
+	}
+}( function( $ ) {
+
+var formResetHandler = function() {
+		var form = $( this );
+		setTimeout( function() {
+			form.find( ".ui-checkboxradio" ).checkboxradio( "refresh" );
```

Comment:
Sounds like that deserves a support comment, or at least a generic comment that the widget doesn't support css-only checkboxes.